### DISCUSSION TITLE
Best way to update self hosted SigNoz

### DISCUSSION THREAD

**Discussion by TheShubhendra** [SigNoz/signoz#4965] [Q&A]
RIght not what I am doing is `git stash` > pull the code > checkout to latest tag, then update the `docker-compose.yaml` run docker/install.sh
But I do not want to keep updating the configuration files in every update.
Should i make copy of the config files and restore them after every update ? or any better way?

**Comment by jansorg** [comment]
I'm pretty new to Signoz, but I created a branch, made and committed my changes to docker-compose.yml and do a `git rebase v0.47.0` or similar to update to a new version.
